README.md

```diff
@@ -220,7 +220,7 @@ Default: `0.8`
 Only assets that compress better than this ratio are processed (`minRatio = Compressed Size / Original Size`).
 Example: you have `image.png` file with 1024b size, compressed version of file has 768b size, so `minRatio` equal `0.75`.
 In other words assets will be processed when the `Compressed Size / Original Size` value less `minRatio` value.
-You can use `1` value to process all assets.
+You can use `1` value to process assets that are smaller than the original.  Use a value of `999` to process all assets even if they are larger than the original (useful when you are pre-zipping all assets for AWS)
```

Comment:
Maybe better `MATH.MAX_SAFE_INTEGER`?

Reply:
Lol, sure, why not :) . (Number.MAX_SAFE_INTEGER)